src/histogram.js

```diff
@@ -259,9 +266,20 @@ angular.module('grafana.directives').directive('grafanaHistogram', function($roo
           }
         };
 
+        var scopedVars = ctrl.panel.scopedVars;
+        var bucketSize = !panel.bucketSize && panel.bucketSize !== 0 ? null : parseFloat(ctrl.templateSrv.replaceWithText(panel.bucketSize.toString(), scopedVars));
+        var minValue = !panel.minValue && panel.minValue !== 0 ? null : parseFloat(ctrl.templateSrv.replaceWithText(panel.minValue.toString(), scopedVars));
+        var maxValue = !panel.maxValue && panel.maxValue !== 0 ? null : parseFloat(ctrl.templateSrv.replaceWithText(panel.maxValue.toString(), scopedVars));
+
+        switch(panel.bucketMode) {
+          case 'count':
+            bucketSize = (maxValue - minValue) / bucketSize;
```

Comment:
It will be later replaced with 1.
But I think there is another problem.
If user set minValue > maxValue, then bucketSize will be negative